7_react/react1/week1/ToDoList/src/Components/TaskInput.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import "../index.css";
+
+function TaskInput() {
+  const [toDoList, setToDoList] = useState([]);
+  const [todo, setTodo] = useState("");
+
+  console.log(todo);
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    const newTodo = {
+      id: new Date().getTime(),
+      text: todo,
+      completed: false,
+    };
+    setToDoList([...toDoList].concat(newTodo));
+    setTodo("");
+  }
+
+  function deleteTodo(id) {
+    const updateToDoList = [...toDoList].filter((todo) => todo.id !== id);
```

Comment:
I think using a spread operator is unnecessary here, you can do just `toDoList.filter((todo) => todo.id !== id);` :)